### PR TITLE
Changed module name in ramp.yml

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

**Review by ashtul — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,9 +18,9 @@ OPTION(USE_TSAN "Use ThreadSanitizer (clang)" OFF)
 OPTION(USE_REDIS_ALLOCATOR "Use redis allocator" ON)
 OPTION(USE_COVERAGE "Build with coverage" OFF)
 
-IF (RS_MODULE_NAME) 
-    ADD_DEFINITIONS("-DREDISEARCH_MODULE_NAME=\"${RS_MODULE_NAME}\"")
-ENDIF()
+# IF (RS_MODULE_NAME)
```
Comment (MeirShpilraien) [inline CMakeLists.txt:21]: @rafie if think we need it for the light build no?

File: src/c_utils/apibench.cpp [hunk 2]
```diff
@@ -19,11 +19,12 @@ extern "C" {
 
 static int my_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 
-  if (RedisModule_Init(ctx, "ft", REDISEARCH_MODULE_VERSION, REDISMODULE_APIVER_1) ==
+  if (RedisModule_Init(ctx, REDISEARCH_MODULE_NAME, REDISEARCH_MODULE_VERSION, REDISMODULE_APIVER_1) ==
```
Comment (MeirShpilraien) [inline src/c_utils/apibench.cpp:22]: Where is it set if its not defined?

File: CMakeLists.txt (hunk 1 above)
Comment (rafie) [reply CMakeLists.txt:21]: Yes, I already fixed it. You know how much I love the way we build Light, right? :-)

File: src/c_utils/apibench.cpp (hunk 2 above)
Comment (rafie) [reply src/c_utils/apibench.cpp:22]: It has a default of "ft" in `version.h`. I'm going to make some changes in this, as I don't want have too granular control in the compiler arguments (like, I just want to declare what we're building with the compiler options, and figure out the details in `version.h`).

**Comment by rafie** [comment]
Some changes are coming here to the packing mechanism, and corresponding changes in RSCoordinator, too.

**Comment by rafie** [comment]
Here's a resulting module link: http://redismodules.s3.amazonaws.com/redisearch-oss/snapshots/redisearch-oss.Linux-rha-x86_64.rafi-modulename1-snapshot.zip

**Review by MeirShpilraien — APPROVED** [review]

**Pull request merged**